packages/sales-api-service/src/server/routes/__tests__/authenticate.spec.js

```diff
@@ -59,6 +59,32 @@ describe('authenticate handler', () => {
       })
     })
 
+    it('authenticates a renewal request if not concessions are returned', async () => {
+      executeQuery.mockResolvedValueOnce([
+        {
+          entity: MOCK_EXISTING_PERMISSION_ENTITY,
+          expanded: {
+            licensee: { entity: MOCK_EXISTING_CONTACT_ENTITY, expanded: {} },
+            concessionProofs: [],
+            permit: { entity: MOCK_1DAY_SENIOR_PERMIT_ENTITY, expanded: {} }
+          }
+        }
+      ])
+      const result = await server.inject({
+        method: 'GET',
+        url: '/authenticate/renewal/CD379B?licenseeBirthDate=2000-01-01&licenseePostcode=AB12 3CD'
+      })
+      expect(permissionForLicensee).toHaveBeenCalledWith('CD379B', '2000-01-01', 'AB12 3CD')
+      expect(result.statusCode).toBe(200)
+      expect(JSON.parse(result.payload)).toMatchObject({
+        permission: expect.objectContaining({
+          ...MOCK_EXISTING_PERMISSION_ENTITY.toJSON(),
+          licensee: MOCK_EXISTING_CONTACT_ENTITY.toJSON(),
+          permit: MOCK_1DAY_SENIOR_PERMIT_ENTITY.toJSON()
+        })
+      })
```

Comment:
please can you separate the three expect assertions into three separate tests?

Reply:
cool, will do